pkg/backingstore/backingstore.go

```diff
@@ -892,6 +892,9 @@ func RunStatus(cmd *cobra.Command, args []string) {
 			if secret.Namespace == "" {
 				secret.Namespace = backStore.Namespace
 			}
+			if secret.Name == "" {
```

Comment:
Just think that maybe on pvPool we should skip printing the secret instead of this fix - as it not really relevant for the user.
if backStore.Spec.Type == nbv1.StoreTypePVPool -> skip secret print

Reply:
Ok.
Hopefully, the update should do the trick.